src/config/dynamic-theme-fixes.config

```diff
@@ -2541,6 +2541,25 @@ developer.android.com
 INVERT
 .devsite-site-logo
 
+CSS
+[theme=android-theme] {
+    --devsite-header-color-upper: var(--darkreader-neutral-background);
+    --devsite-footer-background: var(--darkreader-neutral-background);
+    --devsite-search-form-background-active: var(--darkreader-neutral-background);
+    --devsite-searchbox-inactive: unset;
+    --devsite-searchbox-active: var(--darkreader-neutral-background);
+    --devsite-searchbox-hover: unset;
+}
+.devsite-wrapper {
```

Comment:
For the following 3 CSS entries. `Why?`

Reply:
Let's take https://developer.android.com/jetpack/compose/layout as an example:
1. `.devsite-wrapper` is for the strange background color change across the page. (Notice the gray background divider in the middle of the page.)
<table>
        <tr>
            <td align=center>Origin</td>
            <td align=center>Dark Reader</td>
            <td align=center>Fix</td>
        </tr>
        <tr>
            <td>
<img alt="image" src="https://user-images.githubusercontent.com/17001575/113427256-4a781e00-9407-11eb-97c0-6528f09802c9.png">
</td>
            <td><img alt="image" src="https://user-images.githubusercontent.com/17001575/113417658-750dab00-93f6-11eb-980a-322b592007ca.png"></td>
            <td><img alt="image" src="https://user-images.githubusercontent.com/17001575/113417785-c9188f80-93f6-11eb-9275-23c4a2f84404.png"></td>
        </tr>
</table>

2. `button` background (Notice the thumb up and down icon button as well as the setting button.)
<table>
        <tr>
            <td align=center>Origin</td>
            <td align=center>Dark Reader</td>
            <td align=center>Fix</td>
        </tr>
        <tr>
            <td>
<img alt="image" src="https://user-images.githubusercontent.com/17001575/113427736-28cb6680-9408-11eb-9479-c46bcdf05444.png"></td>
            <td>
<img alt="image" src="https://user-images.githubusercontent.com/17001575/113427859-616b4000-9408-11eb-908a-7d178cf80660.png"></td>
            <td>
<img alt="image" src="https://user-images.githubusercontent.com/17001575/113427916-7a73f100-9408-11eb-994a-b8f6c86f6860.png"></td>
        </tr>
</table>

3. `code` background (Notice the inline code background)
I have no idea why Dark Reader do not convert the background color for `code` block, and the color `silver` may not be the best color?
<table>
        <tr>
            <td align=center>Origin</td>
            <td align=center>Dark Reader</td>
            <td align=center>Fix</td>
        </tr>
        <tr>
            <td>
<img alt="image" src="https://user-images.githubusercontent.com/17001575/113428325-2ddce580-9409-11eb-9c52-ea7d6f51fbc7.png">
</td>
            <td>
<img alt="image" src="https://user-images.githubusercontent.com/17001575/113428272-1998e880-9409-11eb-8204-a4192b93c1c1.png">
</td>
            <td>
<img alt="image" src="https://user-images.githubusercontent.com/17001575/113428198-f2421b80-9408-11eb-8936-7854872223d6.png"></td>
        </tr>
</table>